templateMaker/Makefile

```diff
@@ -0,0 +1,48 @@
+GCC=g++
```

Comment:
This will need update to include the correct paths to append framework interface directory here. Have a look at the makefile for analysisOnData